pkg/handlers/upload.go

```diff
@@ -141,10 +143,20 @@ func (h *Handler) UploadExistingApp(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if err := preflight.Run(a.ID, a.Slug, newSequence, a.IsAirgap, archiveDir); err != nil {
-		logger.Error(err)
-		w.WriteHeader(500)
-		return
+	if !uploadExistingAppRequest.SkipPreflights {
+		if err := preflight.Run(a.ID, a.Slug, newSequence, a.IsAirgap, archiveDir); err != nil {
+			logger.Error(err)
+			w.WriteHeader(500)
+			return
+		}
+	}
+
+	if uploadExistingAppRequest.Deploy {
+		if err := version.DeployVersion(a.ID, newSequence); err != nil {
+			logger.Error(errors.Wrap(err, "failed to deploy latest version"))
```

Comment:
This was based on https://github.com/replicatedhq/kots/blob/v1.33.2/pkg/updatechecker/updatechecker.go#L259, but it's the only place in this file that uses `errors.Wrap`. So not sure if it makes sense to include.